docs/pages/preview/moderated-sessions.mdx

```diff
@@ -0,0 +1,192 @@
+  name: prod-access
+spec:
+  allow:
+    kubernetes_groups:
+      - system:masters
+    require_session_join:
+      - name: Audtor oversight
+        filter: 'contains(user.roles, "auditor")'
+        kinds: ['k8s']
+        modes: ['moderator']
+EOF
+
+tctl create -f prod-access.yaml
+```
+
+### Step 2/4: Create the auditor role
+
+```sh
+cat > auditor.yaml <<EOF
+kind: role
+metadata:
+  name: auditor
+spec:
+  allow:
+    join_sessions:
+    - kinds:
+      - k8s
+      modes:
+      - moderator
+      name: Auditor
+      roles:
+      - member
+EOF
+
+tctl create -f auditor.yaml
+```
+
+### Step 3/4: Add users
+
+```sh
+tctl users add adam --roles=prod-access
+tctl users add bob --roles=auditor
+```
+
+### Step 4/4: Try it out
+
+```sh
+# adam
+
+tsh kube login minikube
+tsh kube exec -st nginx /bin/bash
+```
+
+```sh
+# bob
+
+tsh kube join $SESSION_ID
+```
+
+## RBAC
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and whom are required to be present to start one.
+
+The system is based around *require policies* and *allow policies*. Require policies define a set of conditions
+that must be a met for a session to start/run. The allow policies that matter for a given session
+are always those of user that initiated the session. Allow policies are used to define
+what users a session may join and under what conditions.
+
+### Participant Modes
+
+A participant joining a session will always have one of three modes:
+- Peers can join and collaborate in a session. They can view output and send input.
+- Moderators can join and watch a session, they can view output and forcefully terminate the session at will.
+- Observers can join and watch a session. They cannot control the session in any way.
+
+When joining a session with `tsh join` or `tsh kube join`, a mode can be specified with `--mode <mode>`
+where mode is one of `peer`, `moderator` or `observer`. By default, the mode is `peer` for SSH and
+`moderator` for Kubernetes sessions.
+
+A participant may leave a session with the shortcut `c` while in observer or moderator mode.
+When in moderator mode, a participant may also forcefully terminate the session at any point in time
+with the shortcut `t`.
+
+### Session kinds
+
+Require and allow policies have to specify which sessions they apply to. Currently, only `ssh` and `k8s`
+are valid alternatives.
+
+### Filters
+
+Require policies can specify which users they consider as valid with a filter expression. They work like
+other filters used within Teleport such with access requests. The filter context has a `user` object defined
+which the set fields `roles`, `traits` and `name`.
+
+### Backwards compatibility with Server Access
+
+Previously, Server Access did not have access control for whom can join a session.
+To work around this, RBAC are ignored for users that only have V4 roles.
+New and modified roles are created as V5 roles any having any attached V5 role will
```

Comment:
I had trouble following this sentence. Could you please write it in a different way?